src/app/features/reporting/report-config.ts

```diff
@@ -24,15 +24,46 @@ class ReportConfig extends Entity {
   @DatabaseField() mode?: string;
 
   /**
-   * (sql only) list of arguments needed for the sql query. Placeholder "?" will be replaced.
+   * version of ReportConfig syntax. Just relevant for SqlReports
+   */
+  @DatabaseField() version?: number = 1;
+
+  /**
+   * (sql v1 only) list of arguments needed for the sql query. Placeholder "?" will be replaced.
    */
   @DatabaseField() neededArgs?: string[] = [];
 
-  /** the definitions to calculate the report's aggregations */
+  /** (reporting/exporting only, in browser reports) the definitions to calculate the report's aggregations */
   @DatabaseField() aggregationDefinitions: any[] = [];
 
-  /** (sql only) the definition to calculate the report */
+  /** (sql v1 only) the definition to calculate the report */
   @DatabaseField() aggregationDefinition: string | undefined = undefined;
+
+  /**
+   *  (sql v2 only) transformations that are applied to input variables (e.g. startDate, endDate)
+   *  example: {startDate: ["SQL_FROM_DATE"], endDate: ["SQL_TO_DATE"]}
```

Comment:
```suggestion
   *  (sql v2 only) transformations that are applied to input variables (e.g. startDate, endDate)
   * the keys can be used as placeholders (`$key`) in SQL queries
   *  example: {startDate: ["SQL_FROM_DATE"], endDate: ["SQL_TO_DATE"]}
```

Is this the correct format or ${key}?

Reply:
It's just $key. e.g. 
```sql
SELECT * FROM WHERE c.date BETWEEN $startDate AND $endDate
```